### PR TITLE
Improved coverage configuration

### PR DESCRIPTION
**Pull request by MarkusH**

### CODE REVIEW

File: tests/.coveragerc [hunk 1]
```diff
@@ -1,8 +1,16 @@
 [run]
-omit = */django/contrib/*/tests*,*/django/utils/autoreload.py
+branch = True
+omit =
+    */django/contrib/*/tests*
```
Comment (timgraham) [inline tests/.coveragerc]: I don't think this line is relevant anymore as tests have been moved out of contrib.

File: tests/.coveragerc [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+    */django/utils/autoreload.py
+parallel = True
+source = django
 
 [report]
 ignore_errors = True
+omit =
+    */django/conf/locale/*
```
Comment (timgraham) [inline tests/.coveragerc]: is this to ignore formats.py?

**Comment by timgraham** [comment]
Maybe you could explain in the commit message exactly how things are improved? (maybe it's self-explanatory to someone more familiar with coverage)

File: tests/.coveragerc (hunk 1 above)
Comment (MarkusH) [inline tests/.coveragerc]: Oh, good point. I copied that from the jenkins coverage job.

File: tests/.coveragerc (hunk 2 above)
Comment (MarkusH) [inline tests/.coveragerc]: Yes

**Comment by MarkusH** [comment]
Done. Maybe somebody else with coverage experience can have a look, too.

**Pull request merged**